src/prompts/alert/index.tsx

```diff
@@ -0,0 +1,74 @@
+import './alert.scss';
+
+import { Component, h } from 'preact';
+import { PromptUiProps } from "../ui";
+
+export class Alert extends Component<PromptUiProps, never> {
+  onRequestNativePrompt = () => {
+      this.props.onPromptAccepted(this.props.config);
+  };
+
+  onRequestCancel = () => {
+      this.props.onPromptDeclined(this.props.config);
+  };
+
+  render() {
+      const classes = `kumulos-prompt kumulos-prompt-${this.props.promptManagerState} kumulos-alert-container`;
+      const config = this.props.config;
+
+      if (config.type !== 'alert') {
+          return null;
+      }
+
+      const { labels, colors } = config;
+
+      const heading = labels?.alert?.heading;
+      const body = labels?.alert?.body;
+      const iconUrl = config.iconUrl;
+
+      const cancel = labels?.alert?.declineAction ?? 'maybe later';
+      const accept = labels?.alert?.acceptAction ?? 'yes';
+
+      const bgColor = colors?.alert?.bg;
+      const fgColor = colors?.alert?.fg;
+      const declineActionBgColor = colors?.alert?.declineActionBg;
```

Comment:
Same q's here, reduce the optional chains?

Reply:
Done 👍